bindings/core/src/response.rs

```diff
@@ -117,6 +117,9 @@ pub enum Response {
     /// - [`GetBlockMetadata`](crate::method::ClientMethod::GetBlockMetadata)
     BlockMetadata(BlockMetadataResponse),
     /// Response for:
+    /// - [`GetBlockFull`](crate::method::ClientMethod::GetBlockFull)
+    BlockFull(BlockWithMetadataResponse),
```

Comment:
Maybe we should call these `GetBlockWithMetadata` and `BlockWithMetadata` in our bindings.

Reply:
I wasn't sure about the naming, should I also rename the methods on the client to get_block_with_metadata() then?

Reply:
I would